### PR TITLE
Anya/4503 judge name

### PR DESCRIPTION
**Pull request by aroltsch**
connects #4503 

Testing:
1. Ensure queue is working pointing to Vacols UAT
2. Ensure the join works correctly even if there are nil values for decass table
3. Ensure queue is working locally using fakes

Output: 

image

image

### CODE REVIEW

File: app/models/appeal.rb [hunk 1]
```diff
@@ -46,6 +46,10 @@ class UnknownLocationError < StandardError; end
   # These are only set when you pull in a case from the Case Assignment Repository
   attr_accessor :date_assigned, :date_received, :date_completed, :signed_date, :docket_date, :date_due
 
+  # These attributes are needed for the Fakes::QueueRepository.tasks_for_user to work
+  # because it is using an Appeal object
```
Comment (aroltsch) [inline app/models/appeal.rb]: I am not a big fan of this.

Comment (amprokop) [reply app/models/appeal.rb]: Let's fix it! We shouldn't use the `Appeal` model, you're right — could we just make `Fakes::QueueRepository` return an OpenStruct instead?

File: app/models/vacols/case_assignment.rb [hunk 2]
```diff
@@ -60,13 +60,20 @@ def select_tasks
...
+             "decass.deadusr as added_by_css_id",
              "decass.dedeadline as date_due",
+             "decass.deadusr as added_by",
              "folder.tinum as docket_number")
         .joins(<<-SQL)
           LEFT JOIN decass
             ON brieff.bfkey = decass.defolder
-          JOIN staff
-            ON brieff.bfcurloc = staff.slogid
+          FULL JOIN staff s1
```
Comment (amprokop) [inline app/models/vacols/case_assignment.rb]: Hmm would a `LEFT JOIN` work as well?

File: app/models/appeal.rb (hunk 1 above)
Comment (aroltsch) [reply app/models/appeal.rb]: I can!

File: app/models/vacols/case_assignment.rb (hunk 2 above)
Comment (aroltsch) [reply app/models/vacols/case_assignment.rb]: You are right!

File: app/models/appeal.rb (hunk 1 above)
Comment (aroltsch) [reply app/models/appeal.rb]: It will actually require a lot of refactoring to the fakes queue_repository. I can do it in the next PR.

**Pull request merged**